react/lib/components/Widget/AltpaymentWidget.tsx

```diff
@@ -414,7 +415,7 @@ export const AltpaymentWidget: React.FunctionComponent<AltpaymentProps> = props
                       <Select
                         labelId="select-coin-label"
                         className={classes.select_box}
-                        value={selectedCoin?.coin}
+                        value={selectedCoin?.coin ?? null}
```

Comment:
this and the similar line below were put here to fixes console errors that would appear in the browser